components/library/qt4/manifests/sample-manifest.p5m

```diff
@@ -3887,9 +3887,11 @@ file path=usr/lib/qt/4.8/mkspecs/solaris-cc/qmake.conf
 file path=usr/lib/qt/4.8/mkspecs/solaris-cc/qplatformdefs.h
 file path=usr/lib/qt/4.8/mkspecs/solaris-g++-64/qmake.conf
 file path=usr/lib/qt/4.8/mkspecs/solaris-g++-64/qmake.conf.~1~
+file path=usr/lib/qt/4.8/mkspecs/solaris-g++-64/qmake.conf.~2~
 file path=usr/lib/qt/4.8/mkspecs/solaris-g++-64/qplatformdefs.h
 file path=usr/lib/qt/4.8/mkspecs/solaris-g++/qmake.conf
 file path=usr/lib/qt/4.8/mkspecs/solaris-g++/qmake.conf.~1~
+file path=usr/lib/qt/4.8/mkspecs/solaris-g++/qmake.conf.~2~
```

Comment:
I guess the sample-manifest was generated on dirty workspace?

Reply:
Most likely these were copied by the build system.